scripts/src/main/resources/scripts/functions

```diff
@@ -741,6 +741,23 @@ function doUnzip() {
   doRunCommand "unzip -qn '${1}' -d '${target_dir}'"
 }
 
+function doInstallWithPackageManager() {
+  
+  for installString in "${@}"
+  do
+    first="${installString/ */}"
+    
+    if command -v "${first}" > /dev/null
+    then
+      doEcho "Installing: ${installString}"
+      doRunCommand sudo "${installString}"
```

Comment:
`doRunCommand` expected the command as first argument, second one is comment so this is actually wrong.

Reply:
I fixed it so also resolved.